app/views/posts/show.html.erb

```diff
@@ -0,0 +1,48 @@
+<h1><%= @post.title %></h1>
+<h3>Category: <%= @post.category.title %></h3>
```

Comment:
If you are going to have your show page require @post.category to exist you should have your model validate that it exists.